tests/dp_computations_test.py

```diff
@@ -172,6 +172,73 @@ def test_compute_dp_sum(self):
         self._test_gaussian_noise(results, 10, params.eps, params.delta,
                                   l2_sensitivity)
 
+    def test_equally_split_budget(self):
+        # The number of mechanisms must be bigger than 0.
+        with self.assertRaises(ValueError):
+            pipeline_dp.dp_computations.equally_split_budget(0.5, 1e-10, 0)
+
+        # Only one mechanism.
+        self.assertEqual(
+            pipeline_dp.dp_computations.equally_split_budget(0.5, 1e-10, 1),
+            [(0.5, 1e-10)])
+
+        # Multiple mechanisms.
+        expected_budgets = [(0.5 / 5, 1e-10 / 5) for _ in range(4)]
+        expected_budgets.append((0.5 - 4 * (0.5 / 5), 1e-10 - 4 * (1e-10 / 5)))
+
+        self.assertEqual(
+            pipeline_dp.dp_computations.equally_split_budget(0.5, 1e-10, 5),
+            expected_budgets
+        )
+
+    def test_compute_dp_mean(self):
+        params = pipeline_dp.dp_computations.MeanVarParams(
+            eps=0.5,
+            delta=1e-10,
+            low=1,
+            high=10,
+            max_partitions_contributed=1,
+            max_contributions_per_partition=1,
+            noise_kind=pipeline_dp.NoiseKind.LAPLACE)
+
+        (sum_eps, sum_delta), (count_eps,
+                               count_delta) = pipeline_dp.dp_computations.equally_split_budget(
+            params.eps, params.delta, 2)
+        l0_sensitivity = params.l0_sensitivity()
+
+        # Laplace Mechanism
+        results = [
+            pipeline_dp.dp_computations.compute_dp_mean(count=10, sum=100,
+                                                        dp_params=params)
+            for _ in range(1000000)
+        ]
+        count_values, sum_values, mean_values = zip(*results)
```

Comment:
Could you please check that np.mean(mean_values) is close to the non-dp mean and that np.std(mean_values) is small enough. The same for Gaussian

Reply:
Sure! I am not quite sure what value should delta have here. After testing it multiple times, I noticed that the mean might differ by even ~7.0

Reply:
Yeah, this makes sense. 
denominator = 10 + lap(4), when denominator is small, mean is large -> which makes disproportional contribution in np.mean() computation. This is just an example, that DP doesn't work nice with small number of elements (10 here).

Please update sum: 100 -> 10000, count: 10 -> 1000. (count = 1000 are reasonable number when DP becomes applicable)
It should provide reasonable close mean.

Reply:
Thanks for the explanation and sorry for the late response! I have a small follow up question about the standard deviation of sum/mean. As I understand it, the mean is a ratio between two Laplace/Gaussian distributions each one with its own std. I tried to compute the std of the ratio based on the std of numerator and denominator, but I couldn't find a relation. Is there something that I am missing out?

Reply:
Correct, mean is the the ratio of 2 Laplace/Gaussian distributions. Unfortunately it's pretty hard to work with ratio distributions ([wiki](https://en.wikipedia.org/wiki/Ratio_distribution)). 
But often there is no need to work with ratio distributions, for eg. in case of mean, the idea that if the noise is small in comparison to the count, then 
(sum+noise_1)/(count+noise_2) is not far from sum/count.

When the noise is not small in comparison to the count, the noise will distort the results significantly and in such cases DP is not applicable. But that's a fundamental problem with anonymization - it's hard (if possible) to anonymize small datasets. That's why I suggested to change the test data for more suitable for DP data.

Reply:
I changed the test data, and the mean is close enough now. Thanks!